Description: don't write current date to comment field in dvi file
Author: Thorsten Alteholz <debian@alteholz.de>
Last-Update: 2015-05-04

Index: feynmf-1.08/Makefile
===================================================================
--- feynmf-1.08.orig/Makefile	2018-12-07 11:07:14.441536895 +0100
+++ feynmf-1.08/Makefile	2018-12-07 11:08:47.981534091 +0100
@@ -153,9 +153,9 @@
   fmfman209.drv fmfman209ps.drv: feynmf209.sty
 
 feynmf.dvi: feynmf.dtx feynmf.drv feynmf.sty feynmf.mf $(MANPICS)
-	-$(LATEX) $*.drv
+	-$(LATEX) --output-comment="" $*.drv
 	$(RUN_METAFONT)
-	-$(LATEX) $*.drv
+	-$(LATEX) --output-comment="" $*.drv
 	$(RUN_MAKEINDEX)
 	$(RESOLVE_XREF)
 
@@ -167,9 +167,9 @@
 	$(RESOLVE_XREF209)
 
 feynmp.dvi: feynmf.dtx feynmp.drv feynmp.sty feynmp.mp $(MANPICS)
-	-$(LATEX) $*.drv
+	-$(LATEX) --output-comment="" $*.drv
 	$(RUN_METAPOST)
-	-$(LATEX) $*.drv
+	-$(LATEX) --output-comment="" $*.drv
 	$(RUN_MAKEINDEX)
 	$(RESOLVE_XREF)
 
@@ -181,9 +181,9 @@
 	$(RESOLVE_XREF209)
 
 fmfman.dvi: feynmf.dtx fmfman.drv feynmf.sty feynmf.mf $(MANPICS)
-	-$(LATEX) $*.drv
+	-$(LATEX) --output-comment="" $*.drv
 	$(RUN_METAFONT)
-	-$(LATEX) $*.drv
+	-$(LATEX) --output-comment="" $*.drv
 	$(RUN_MAKEINDEX)
 	$(RESOLVE_XREF)
 
@@ -195,9 +195,9 @@
 	$(RESOLVE_XREF209)
 
 fmfmanps.dvi: feynmf.dtx fmfmanps.drv feynmp.sty feynmp.mp $(MANPICS)
-	-$(LATEX) $*.drv
+	-$(LATEX) --output-comment="" $*.drv
 	$(RUN_METAPOST)
-	-$(LATEX) $*.drv
+	-$(LATEX) --output-comment="" $*.drv
 	$(RUN_MAKEINDEX)
 	$(RESOLVE_XREF)
 
